File: src/main/java/com/campus/dormrepair/enums/UserRole.java
package com.campus.dormrepair.enums;

public enum UserRole {
    STUDENT("学生"),
    DORM_MANAGER("宿管"),
    REPAIRMAN("维修工"),
    LOGISTICS_SUPERVISOR("后勤主管");

    private final String label;

    UserRole(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}

File: app/Enums/UserRole.php
<?php

namespace App\Enums;

enum UserRole: string
{
    case Clerk = 'clerk';
    case Approver = 'approver';

    public function label(): string
    {
        return match ($this) {
            self::Clerk => '业务专员',
            self::Approver => '审批负责人',
        };
    }
}
